helios-tools/src/main/java/com/spotify/helios/cli/command/HostListCommand.java

```diff
@@ -107,6 +125,16 @@ int run(final Namespace options, final HeliosClient client, final PrintStream ou
         .from(client.listHosts().get())
         .filter(containsPattern(pattern))
         .toList();
+    HostStatus.Status statusFilter = null;
+    final String statusFilterStr = options.getString(statusArg.getDest());
+    if (!isNullOrEmpty(statusFilterStr)) {
+      try {
+        statusFilter = HostStatus.Status.valueOf(statusFilterStr);
```

Comment:
Ah yes. Because of the argparse `Choices`, if the user uses lowercase, an error will occur before we even get here.